ml-agents/mlagents/trainers/bc/models.py

```diff
@@ -44,9 +44,7 @@ def __init__(
                         size,
                         activation=None,
                         use_bias=False,
-                        kernel_initializer=c_layers.variance_scaling_initializer(
-                            factor=0.01
-                        ),
+                        kernel_initializer=tf_variance_scaling(0.01),
```

Comment:
As mentioned in the PR, I don't love the name. Would like to find a better way to handle this.

Also, the parameter name changed (factor vs scale) to I just left it as positional.